CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-## 0.1.0
+## 0.2.1
+
+* Bump native_assets_cli to 0.10.0.
+
+## 0.2.0
```

Comment:
This is the description for 0.1.0, and new entries are placed at the bottom. Perhaps that is not very conventional. :)

Reply:
Probably not. It means your users have to scroll down a lot to see what's new on https://pub.dev/packages/flutter_gpu_shaders/changelog once you release 100 versions :)